src/main/java/com/example/bookstore/dto/user/UserResponseDto.java

```diff
@@ -0,0 +1,12 @@
+package com.example.bookstore.dto.user;
+
+import lombok.Data;
+
+@Data
+public class UserResponseDto {
```

Comment:
if you start using features from Java 17+ like record, use them everywhere you can